monero-outreach-docs/translations/fr/archive/dandelion_fr.md

```diff
@@ -0,0 +1,67 @@
+
+### Dandelion
+
+Afin d'adresser ce problème, une équipe de chercheurs de l'Université de l'Illinois ont d'abord développé un ensemble de techniques qu'ils ont appelé Dandelion [2]. L'application devait être pour Bitcoin, mais Dandelion s'applique également à Monero. Le principe de Dandelion est de de router, en premier lieu, les transactions à un nœud distant d'une manière spéciale et indétectable avant d'initier la propagation par "flooding".
+
+Les auteurs de Dandelion ont plongé dans les aspects théoriques et pratiques de l'utilisation de botnets pour retracer les sources de transactions de cryptomonnaies. Cet effort a abouti à plusieurs résultats, dont la création de modèles mathématiques pour quantifier et mesurer l'anonymat (voir [2] pour plus d'information), qui ont été utilisés pour analyser trois approches de propagation des transactions sur un réseau de cryptomonnaie: 1) le "flooding" de base (la méthode de Monero), 2) un "flooding" aléatoire, appelé diffusion (la méthode de Bitcoin), et 3) la diffusion par proxy. Cette dernière approche transmet d'abord la transaction à un nœud aléatoire, qui la propage ensuite par la méthode de diffusion. Les trois techniques ont ultimement été jugées inadéquates selon les modèles mathématiques des chercheurs.
+
+Pour adresser cette insuffisance, les auteurs ont proposé Dandelion. Dandelion définit un procédé pour trouver un nœud proxy pour effectuer la transmission, qu'on appelle la phase d'anonymité (ou la phase tige - "stem" en anglais). L'outil établit un second procédé pour l'émission, appelé la phase ouate ("spread" ou "fluff" en anglais). En général, les deux phases font appel à un ensemble de pairs différent, avec la différence marquée que l'ensemble de connexions pour la phase d'anonymité change avec le temps. Le nom Dandelion nous vient du procédé où, dans un premier temps, le système trouve un nœud proxy selon un chemin de recherche particulier et linéaire, puis à partir de ce nœud proxy le système propage l'information rapidement et symétriquement-la forme de la propagation d'information ressemble à une fleur de pissenlit ("Dandelion" en anglais).
```

Comment:
[1] 
Dandelion définit un procédé pour trouver un nœud proxy pour effectuer la transmission, qu'on appelle la phase d'anonymité (ou la phase tige - "stem" en anglais)  
>>>  
Dandelion définit un procédé qu'on appelle la phase d'anonymité (ou la phase tige - "stem" en anglais), afin de trouver un nœud proxy qui effectuera la transmission   

Reason: Reordering and slightly modifying the wording makes it clearer to me

[2]
appelé la phase ouate ("spread" ou "fluff" en anglais)  >>> appelé la phase de dispersion ("spread" ou "fluff" en anglais)   

Reason: I discovered the word 'ouate' by reading this. I think it is indeed a possible translation. But it is I think quite rare (or is specific to some regions?). Therefore, I am not sure that this is the most appropriate term. A more common word, such as 'dispersion' or an adequate translation of 'fluff' (phase duvet? phase coton?) is probably preferable.
Note that this expression 'phase ouate' appears at several occasion over the text. I noted this recommendation only here (once), but obviously whatever is the choice of the translator (keep 'ouate' or change it) the potential modification will need to be propagated all along the text for sake of consistency.

Reply:
I had trouble with this one, and I agree with your recommendation. The analogy in the original text is obviously to the dandelion flower, with a stem and the fluff part that blows away in the wind. The French term for this is indeed ouate, which is why I chose it, but "phase ouate" isn't quite to my liking either. I'm going to integrate your recommendation of "phase de dispersion".